first_hour_on_terra/working_with_data_collections.ipynb

```diff
@@ -0,0 +1,449 @@
+    " style=input_style,\n",
+    " value=output_workspace_id.value\n",
+    ")\n",
+    "widgetLink = widgets.jslink((input_dc_workspace_id,'value'),(input_workspace_id,'value'))\n",
+    "input_short_description = widgets.Text(\n",
+    " placeholder=\"<DESCRIPTION>\",\n",
+    " description=\"Description:\",\n",
+    " style=input_style\n",
+    " )\n",
+    "input_version = widgets.Text(\n",
+    " placeholder=\"<VERSION>\",\n",
+    " description=\"Version:\",\n",
+    " style=input_style\n",
+    " )\n",
+    "display(input_workspace_id)\n",
+    "display(input_short_description)\n",
+    "display(input_version)\n",
+    "\n",
+    "# define a variable that will be reset when the button is pressed\n",
+    "buttonOutput = 'Please provide appropriate values in the input boxes.'\n",
+    "\n",
+    "# define a function for the button to call\n",
+    "def button_click_event(b):\n",
+    "    with output:\n",
+    "        global buttonOutput\n",
+    "        terraCommand = \"\"\"terra workspace set-property \\\\\n",
+    "        --workspace={0} \\\\\n",
+    "        --properties=\"terra-type=data-collection,terra-workspace-short-description={1},terra-workspace-version={2}\"       \"\"\".format(input_workspace_id.value,input_short_description.value,input_version.value)       \n",
+    "        # print(terraCommand)\n",
+    "        os.system(terraCommand)\n",
+    "\n",
+    "# get a reference to the widget output\n",
+    "output = widgets.Output()\n",
+    "\n",
+    "button = widgets.Button(\n",
+    "    description='Convert to data collection',\n",
+    "    disabled=False,\n",
+    "    button_style='',\n",
+    "    tooltip='Click to convert to data collection',\n",
+    "    icon='check',\n",
+    "    layout=Layout(width='50%', height='40px')\n",
+    ")\n",
+    "\n",
+    "#bind the button_click_event to the button call event\n",
+    "button.on_click(button_click_event)\n",
+    "\n",
+    "# show the current state of the output\n",
+    "print(buttonOutput)\n",
+    "\n",
+    "#display the button\n",
+    "display(button, output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Add data collection to workspace\n",
+    "\n",
+    "Follow the steps below to add your new data collection to an Enterprise Terra workspace.<br>The video below provides a visual walkthrough of these steps.\n",
+    "\n",
+    "1. In the [Enterprise Terra workspace UI](https://terra.verily.com/workspaces), select a workspace.\n",
+    "1. Navigate to the Resources tab.\n",
+    "1. Click the \"+ Data catalog\" button.\n",
+    "1. Select your newly created data collection from those listed in the modal.\n",
+    "1. Navigate through the steps in the modal to complete the addition of the data collection to your workspace.\n",
+    "1. On a cloud environment terminal in your workspace, run `terra resource list` to confirm the data collection has been added.\n",
```

Comment:
As an alternative option, we could add a cell here with `!terra resource list` that one could run after adding the data collection.

Reply:
Since I cannot capture that step in the screencast, I'm a little worried it might be awkward to switch back to Bash commands at this point...what do you think? I am happy to include it but not sure where to put it relative to the screencast.

Reply:
I'd also be ok if it said "On a cloud environment terminal in your workspace or in a Jupyter cell". Its a minor point, I don't think it blocks this from being merged.

Reply:
I have added the command as you suggested -- I tested it out and I am no longer concerned it will be awkward after seeing it.

Reply:
Your instructions said `terra resource list` but your new command says `!terra workspace list`. Which should it be? And could you please also add some markdown briefly mentioning what they should be seeing? Since the cell is divided from the instructions by the video the step "Run the cell below" can be missed.

Reply:
Per discussion over chat, I've removed this step.